### ISSUE TITLE
Behaviour of Enter/Return

### ISSUE THREAD

**Issue by afinne** [gangstead/atom-incremental-search#34]
In the old version of the incremental search package, Enter would stop the search and leave the cursor at the currently selected instance. It still states so in the keybindings list for Os X: 
```
    return - stop the search and leave the cursor where it is
    esc - cancel the search and return the cursor to where it was before searching
```
However, the behaviour now is that Enter selects the next instance (just like Cmd-I does). This is also stated in the readme, however, in order to leave the cursor on the currently selected instance, it says:
```
    "When you've found the text you are looking for, press esc 
    to stop the search and leave the cursor on the search result. 
    To cancel the search and return to where you started, press esc."
```
I think that the correct behaviour would be that Enter would stop searching and leave the cursor where it should be, while esc cancels searching and returns to where it started. Cmd-I selects the next instance.

**Comment by nwinkler** [comment]
:+1: Noticed the same earlier today...

Great work otherwise - I'm really happy to see that work is being done on this package!

**Comment by afinne** [comment]
Yes, sorry, forgot to write how happy I am that this gets updated :) Thank you really much :)

**Comment by afinne** [comment]
Did a local change to the installed package. Line 33 in ~/.atom/packages/incremental-search/lib/input-view.coffee says: 
```'core:confirm': => @trigger('forward') # jump to next match, previously did @stopSearch()```

I just changed it back to ```@stopSearch()``` instead of ```@trigger('forward')```

Seems to work.

**Comment by gangstead** [comment]
Fixed in #36

**Comment by gangstead** [comment]
Published `v5.0.2`
